### DISCUSSION TITLE
YouTube playlist of individual sections

### DISCUSSION THREAD

**Discussion by PatrickAlphaC** [Cyfrin/foundry-full-course-cu#213] [Announcements]
Due to popular demand, we've broken down each section into it's own video!

https://www.youtube.com/playlist?list=PL2-Nvp2Kn0FPH2xU3IbKrrkae-VVXs1vk

If you want to watch byte-sized (get it?) videos instead of all 27-hours in those 3 long videos, you can follow along with the playlist here!

https://www.youtube.com/playlist?list=PL2-Nvp2Kn0FPH2xU3IbKrrkae-VVXs1vk

**Comment by ChefAharoni** [comment]
This is **AMAZING** and very helpful, thank you very much!

**Comment by Vinyl-Davyl** [comment]
nice list patrick
